src/signals/incident/components/form/Header/index.js

```diff
@@ -2,41 +2,48 @@ import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import styled, { css } from 'styled-components';
 import { themeSpacing, themeColor } from '@amsterdam/asc-ui';
+import Label from 'components/Label';
 
-const Children = styled.div`
+const Wrapper = styled.div`
   display: flex;
-  flex-flow: column;
-`;
+  flex-direction:column;
 
-const Wrapper = styled.div`
   ${({ invalid }) =>
     invalid &&
     css`
       border-left: ${themeColor('support', 'invalid')} 2px solid;
       padding-left: ${themeSpacing(3)};
     `}
+
+  /* keep a 3 units margin above the last element/control  */
+  & > :last-child :not(& > :first-child) {
```

Comment:
This should be improved: on the incident page of the "type" field (logged in), this increases the margin between between the type explanation and the radio buttons.

Reply:
Solved by grouping the radio buttons and the description. We assume the last element is the container with the space on top.